build.gradle.kts

```diff
@@ -53,53 +55,47 @@ plugins {
     idea
 }
 
-repositories {
-    mavenLocal()
-    standardToSpineSdk()
-}
-
-dependencies {
-    // Depend on Spine Base to grab the Proto options
-    // from `spine/options.proto`.
-    api(Spine.base)
+// It is assumed that every module in the project requires
+// a typical configuration.
+allprojects {
 
-    Protobuf.libs.forEach { implementation(it) }
+    repositories.standardToSpineSdk()
 
-    ErrorProne.apply {
-        errorprone(core)
+    apply {
+        plugin("kotlin")
+        plugin("net.ltgt.errorprone")
+        plugin("detekt-code-analysis")
+        plugin("com.google.protobuf")
+        plugin("io.spine.protodata")
+        plugin("idea")
     }
-}
 
-// Uncomment the following section
-// to add ProtoData plugins:
+    dependencies {
+        api(Spine.base)
 
-/*
-protoData {
-    plugins(
-        "com.acme.MyProtoDataPlugin",
-        "com.bar.AnotherProtoDataPlugin"
-    )
-}
-*/
+        Protobuf.libs.forEach { implementation(it) }
 
-protobuf {
-    protoc {
-        artifact = Protobuf.compiler
+        ErrorProne.apply {
+            errorprone(core)
+        }
     }
-}
 
-object BuildSettings {
-    private const val JAVA_VERSION = 11
+    protobuf {
+        protoc {
+            artifact = Protobuf.compiler
+        }
+    }
 
-    val javaVersion: JavaLanguageVersion = JavaLanguageVersion.of(JAVA_VERSION)
+    // Apply a typical configuration to every module.
+    applyConfiguration()
 }
 
 /**
  * The alias for typed extensions functions related to modules of this project.
  */
 typealias Module = Project
 
-project.run {
+fun Module.applyConfiguration() {
     configureJava()
     configureKotlin()
     setupTests()
```

Comment:
I think, that's a typo in the original PR. It should be `setUpTests` (where "set up" is a verb), not `setup` (where "setup" is a noun).

Reply:
Done.